### PR TITLE
fix: [CCM-5435]: fix efficiency score -1 when data is not present

### PR DESCRIPTION
**Pull request by rajarshee-harness**
##### Jira Links:

https://harness.atlassian.net/browse/CCM-5435

##### Screenshots:

Screenshot 2022-03-08 at 11 16 01 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rajarshee-harness** [comment]
trigger rebase

**Comment by rajarshee-harness** [comment]
trigger rebase

**Comment by rajarshee-harness** [comment]
retrigger jest

**Review by jenil-harness — APPROVED** [review]

File: src/modules/75-ce/common/CostTrend.tsx [hunk 1]
```diff
@@ -6,18 +6,26 @@
...
-import { Color, IconName, Text } from '@wings-software/uicore'
+import { Color, IconName, Text, FontVariation } from '@wings-software/uicore'
 
 interface TrendProps {
   value: number
   downIcon?: string
   upIcon?: string
   iconSize?: number
   flipColors?: boolean
+  defaultIcon?: string
```
Comment (rishik30) [inline src/modules/75-ce/common/CostTrend.tsx:17]: should we have `IconName` as type here?

Comment (rishik30) [reply src/modules/75-ce/common/CostTrend.tsx:17]: also, is `defaultIcon` a good naming convention?

**Review by rishik30 — APPROVED** [review]

File: src/modules/75-ce/common/CostTrend.tsx (hunk 1 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/common/CostTrend.tsx:17]: > should we have IconName as type here?

`upIcon` and `downIcon` had string as type so went with the same for defaultIcon.

> also, is defaultIcon a good naming convention?

The icon will only show when data is not present so named it `defaultIcon`.

**Pull request merged**